open-metadata-resources/open-metadata-labs/administration-labs/understanding-platform-services.ipynb

```diff
@@ -507,7 +507,7 @@
  ],
  "metadata": {
   "kernelspec": {
-   "display_name": "Python 3",
+   "display_name": "Python 3 (ipykernel)",
```

Comment:
I am not sure why we need this change?

Reply:
It is not something I changed myself. 
Not sure how I would have to change it

Reply:
In the docker build for jupyter we always retrieve the latest version

FROM docker.io/jupyter/base-notebook:latest